core/src/main/java/com/g16/feyrune/Util/Random.java

```diff
@@ -0,0 +1,5 @@
+package com.g16.feyrune.Util;
+
+public class Random {
+    public static final java.util.Random random=new java.util.Random();
+}
```

Comment:
Should be using a public interface to interact with random instead

Reply:
Is somewhat fixed now